src/main/java/com/springbootWithJPA/springBootRestExample/Employee.java

```diff
@@ -1,56 +1,17 @@
 package com.springbootWithJPA.springBootRestExample;
 
+import lombok.Data;
+
 import javax.persistence.*;
```

Comment:
avoid using wild card imports

Reply:
addressed in https://github.com/Jashika/springBootRestExample/pull/2/commits/0d318196ef3325c1a36a86cd1cdc1a38d93c5ccd